GCRCatalogs/catalog_configs/cosmoDC2_v1.1.4_small_combined_agn.yaml

```diff
@@ -0,0 +1,8 @@
+subclass_name: agncombined.AGNCombinedCatalog
```

Comment:
This whole config file can be written as:

```yaml
based_on: cosmoDC2_v1.1.4_small_composite_agn
suppress_overwrite: ['ra', 'redshift', 'is_central', 'halo_mass', 'galaxy_id', 'dec', 'stellar_mass']
```

to avoid repetition.

Reply:
Oops, I agreed to this but based on an earlier conversation I had already removed cosmoDC2_v1.1.4_small_composite_agn in favor of cosmoDC2_v1.1.4_small_combined_agn.  I don't have a strong preference, but I think it would be less confusing to stick with that earlier change and just use the reader for the combined catalog rather than having a composite catalog available as well.

Reply:
OK, let's remove these configs that are not needed then.